testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-delta-lake-kerberized-hdfs/delta.properties

```diff
@@ -10,3 +10,4 @@ hive.hdfs.authentication.type=KERBEROS
 hive.hdfs.impersonation.enabled=false
 hive.hdfs.presto.principal=hdfs/hadoop-master@LABS.TERADATA.COM
 hive.hdfs.presto.keytab=/etc/hadoop/conf/hdfs.keytab
+delta.register-table-procedure.enabled=true
```

Comment:
needed?

Reply:
Yes, Needed for `io.trino.tests.product.deltalake.BaseTestDeltaLakeHdfsReads#testReads`